app/master/build.py

```diff
@@ -471,19 +473,21 @@ def _perform_async_postbuild_tasks(self):
         data. This method also transitions the FSM to finished after the postbuild tasks are complete.
         """
         try:
-            self._create_build_artifact()
+            timing_data = self._read_subjob_timings_from_results()
+            self._create_build_artifact(timing_data)
             self._delete_temporary_build_artifact_files()
+            self._update_prometheus_metrics()
```

Comment:
No reason. I don't want it to be after the state_machine.trigger() event. That event puts the build in the finished state, and as far as I can tell right now nothing happens to a build after it finishes. I'd like to keep it that way if I can.